src/main/java/org/carlmontrobotics/robotcode2023/RobotContainer.java

```diff
@@ -4,47 +4,67 @@
 
 package org.carlmontrobotics.robotcode2023;
 
+import org.carlmontrobotics.robotcode2023.Constants.OI.Driver;
+import org.carlmontrobotics.robotcode2023.commands.IntakeRoller;
+import org.carlmontrobotics.robotcode2023.commands.OuttakeRoller;
+import org.carlmontrobotics.robotcode2023.subsystems.Roller;
+import org.carlmontrobotics.robotcode2023.subsystems.Roller.RollerMode;
+
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj.PowerDistribution;
-import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 
 public class RobotContainer {
 
   public final Joystick driverController = new Joystick(0);
   public final Joystick manipulatorController = new Joystick(1);
   public final PowerDistribution pd = new PowerDistribution();
 
+  public final Roller roller = new Roller();
+
   public RobotContainer() {
     configureButtonBindingsDriver();
     configureButtonBindingsManipulator();
   }
 
-  private void configureButtonBindingsDriver() {}
+  private void configureButtonBindingsDriver() {
```

Comment:
Are these buttons supposed to be on the driver controller or the manipulator controller?

Reply:
You're right, they should be on the manipulator. I've moved them over, but keep in mind that this is still a testing configuration. The officially proposed OI is worth a PR in and of itself.